### DISCUSSION TITLE
Examples of Server or sending Color data

### DISCUSSION THREAD

**Discussion by gjermshus** [PlummersSoftwareLLC/NightDriverStrip#79] [Ideas]
Hi,
Great project. Are there any (or any plans) of some example code of sending color data? I would like to learn more about that part of the system and an example would be great. Also to learn how this functionality can be used to create interesting patterns and perhaps connect it to other automation elements/projects.

**Comment by xcalibur839** [comment]
I've been playing around with this as well and have a very bare bones example available here: [https://github.com/xcalibur839/NightDriverController](https://github.com/xcalibur839/NightDriverController)

**Comment by robertlipe** [comment]
This was added after this question was asked, but https://github.com/PlummersSoftwareLLC/NightDriverStrip/discussions/186 gives examples of how to deliver pixel data over wifi to the network-enabled builds of NDS.

**Comment by davepl** [comment]
There’s a Python example called “videoserver.py” that takes the bits from a video and sends them as a color data array.  So that code could do any LED array you could draw, in theory.

- Dave
